src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,24 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  private Uuid parseResult;
+
+  public ServerInfo() {
+    try {
+      parseResult = Uuid.parse(SERVER_VERSION);
+    } catch (IOException exception) {
+      exception.getMessage();
```

Comment:
You're not actually doing anything with the exception. Just set this directly to `this.version` and remove the `try catch`.